### PR TITLE
[DataTable] Use relative path

### PR DESCRIPTION
**Pull request by AndrewMusgrave**
### WHY are these changes introduced?
Causes build to fail

### WHAT is this pull request doing?
Uses a relative import

### How to 🎩
TBD: Not totally sure. Possibly run web in production?

### CODE REVIEW

**Comment by danrosenthal** [comment]
@GoodForOneFare, do you have any advice on tophatting this to make sure it won't cause the same problem that led to https://github.com/Shopify/web/pull/8518?

File: UNRELEASED.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Bug fixes
 
+- Fix selector import in `DataTable` and `Cell` ([#638](https://github.com/Shopify/polaris-react/pull/638))
```
Comment (solonaarmstrong) [inline UNRELEASED.md]: Fixed

Comment (danrosenthal) [reply UNRELEASED.md]: This feels like a great opportunity to use the code suggestion feature.

```suggestion
- Fixed selector import in `DataTable` and `Cell` ([#638](https://github.com/Shopify/polaris-react/pull/638))
```

Comment (solonaarmstrong) [reply UNRELEASED.md]: Oh right, I have to rewire my brain to remember to use it.

**Comment by danrosenthal** [comment]
@GoodForOneFare, also curious if you have any insight as to why this didn't come up earlier in the web CI process? Why didn't this fail until it was in Shipit?

**Comment by GoodForOneFare** [comment]
shrink-ray is temporarily disabled.  It usually catches these problems.

**Pull request merged**